common/src/main/java/com/wynntils/models/mobtotem/MobTotemModel.java

```diff
@@ -74,6 +75,16 @@ public void onWorldChange(WorldStateEvent e) {
     }
 
     public List<MobTotem> getMobTotems() {
-        return mobTotems.values().stream().toList();
+        return mobTotems.values().stream()
```

Comment:
This will reorder mob totems when the player moves around. Is this intentional? (I don't think that is how the old code works, but then again I don't think I've ever seen it with two concurrent mob totems)

Reply:
Oh, yes, this changes stuff. But that case is rare, if not disallowed by Wynn. Anyways, I changed this to `LinkedHashMap` to always have some kind of stable order, what I wanted to achieve with this sort.

Reply:
I was more thinking like if you sort on the totem's creator's name, or its location, then it would be stable independent on the player's current location.

Reply:
(Totem owner is probably easiest to sort on, has a well-defined natural order)